### DISCUSSION TITLE
Welcome to contour Discussions!

### DISCUSSION THREAD

**Discussion by christianparpart** [contour-terminal/contour#147] [General]
## 👋 Welcome!

Many thanks for your interest in this project. This software is entirely built from scratch and mostly developed live on Twitch at https://twitch.tv/christianparpart/

We want to use this forum for general discussions all related directly or indirectly to this project.

Please:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Looking forward to a great time,
Christian Parpart.

**Comment by data-man** [comment]
It would be nice if members could edit the Wiki. :wink:

**Reply by christianparpart** [reply]
So let's do this. (EDIT: ok, members, not strangers, lol :D )

**Reply by christianparpart** [reply]
should be possible now. I've actually enabled write access to all members, i.e. apart from wiki editing, you can now also push to all repos (I'd still like to review changes PRs). :)

**Reply by data-man** [reply]
Thank you!
